packages/grid/_modules_/grid/components/cell/GridEditInputCell.tsx

```diff
@@ -60,18 +67,13 @@ function GridEditInputCell(props: GridRenderEditCellParams & Omit<InputBaseProps
   const ownerState = { classes: rootProps.classes };
   const classes = useUtilityClasses(ownerState);
 
-  const debouncedSetEditCellValue = React.useMemo(
-    () => debounce(api.setEditCellValue, SUBMIT_FILTER_STROKE_TIME),
-    [api.setEditCellValue],
-  );
-
   const handleChange = React.useCallback(
     (event) => {
       const newValue = event.target.value;
       setValueState(newValue);
-      debouncedSetEditCellValue({ id, field, value: newValue }, event);
+      api.setEditCellValue({ id, field, value: newValue, debounceMs }, event);
```

Comment:
If we remove `api` from the `GridCellParams` in v6 maybe we should avoid using it in our components and use `useGridApiContext`

Non-blocking for this PR

Reply:
I'll do in another PR. There're a few components also using the API from the `api` prop.